src/handlers/collect-linked-pulls.ts

```diff
@@ -0,0 +1,78 @@
+import { parseGitHubUrl } from "../helpers/github-url";
+import { Context } from "../types/context";
+import { GitHubLinkEvent, GitHubTimelineEvent, isGitHubLinkEvent } from "../types/github-types";
+
+export type IssueParams = ReturnType<typeof parseGitHubUrl>;
+
+export async function collectLinkedPullRequests(context: Context, issue: IssueParams) {
+  const onlyPullRequests = await collectLinkedPulls(context, issue);
+  return onlyPullRequests.filter((event) => {
+    if (!event.source.issue.body) {
+      return false;
+    }
+    // Matches all keywords according to the docs:
```

Comment:
This is an interesting approach. I think it's fine. 

Should we only support issue.body (the issue specification) or should we also support comments? 

For example, within a separate discussion I could write a comment that says resolves #0 and have the comment itself (and the subsequent conversation) indicate that it's associated with the project? 

I feel like there were examples of this in the past but I can't remember any in particular right now as a reference point.

Reply:
It's a good idea, but I think it is very easy to abuse as anyone could link any comment to get credit on it. Would be worth thinking about it and see if we can make something reliable.

Reply:
Abuse is easy to defend against. We simply delete the offending comment before permit generation.